pkgs/create-neon/dev/expect.ts

```diff
@@ -11,7 +11,7 @@ function readChunks(input: Readable): Readable {
   });
   input.on("close", () => {
     output.write(decoder.end());
-    output.destroy();
+    output.end();
```

Comment:
The `.destroy()` call happened to work in the past but wasn't the right method to use, and it broke in more recent versions of Node.